docs/admin/config/batch_changes.mdx

```diff
@@ -344,12 +344,37 @@ However, if the user deletion is permanent, deleting both account and data, then
 
 ## Batch Spec Library
 
-<Callout type="note"> Batch Spec Library is currently in Experimental.</Callout>
+The Batch Spec Library accelerates batch change adoption by offering curated examples and guided templates that make large-scale code modifications accessible to developers at every skill level.
 
-The Batch Spec Library is a collection of Batch Specs that can be used to create Batch Changes. Sourcegraph provides a few Batch Specs out of the box.
+The library distinguishes between [templates](#templates) and [library examples](#library-examples), where library examples are complete batch specs and templates are library examples with variables.
+
+Sourcegraph instances come with a couple of library examples out of the box. You can use the GraphQL APIs to [manage the Batch Spec Library](#managing-the-batch-spec-library).
+
+As a site admin, you can [feature records](#featured-records) to highlight the most useful examples for your organization.
```

Comment:
Can you do this w/templates too?

And if you like a template, can you turn it into a canonical example when the variables are filled out?

Reply:
Yeah! I mean, this is tangential to this PR of course, but I think that'd be awesome! Like, look, I created a dope batch spec, lemme put that into the canonical lib of examples for my org (given some review)